 #Sensor Parameters
 sensor_parameters:

  # Sensor settings - OS1-16 Sensor
  V_SCAN: 32
  H_SCAN: 512
  groundScanInd: 7
  opening_vertical_angle: 90.0
  opening_horizon_angle: 360.0
  start_angle_v: 45.0
  start_angle_h: -180.0

